### ISSUE TITLE
Way to select codec ?

### ISSUE THREAD

**Issue by mouf13** [armackey/nativescript-twilio-video#3]
HI, 

On Web, to make Twilio work both on ios and chrome, we need to change the preferred codec option.
Could you add this this ? 
Here an example for web :

`connect(token, {
      name: 'room',
      preferredVideoCodecs: [
        { codec: 'VP8', simulcast: true }
        // this._codec === 'VP8' ? { codec: 'VP8', simulcast: true } : { codec: 'H264' }
      ]
    })`

Thanks and thx a lot for your plugin !

**Comment by praveshkhatana** [comment]
Hey @mouf13, 

Did you find solution?

**Comment by armackey** [comment]
@mouf13 I'll have to take a look over the weekend if I have free time.

**Comment by mouf13** [comment]
Hi @armackey & @praveshkhatana,
The options were commented in android/twilio-video & ios/twilio-video.
I also updated the Twilio SDKs.
Unfortunately i can't make the video works between web and Android.
It's ok with IOS.
I can share my updates if need.

**Comment by praveshkhatana** [comment]
@mouf13, yes i also found the same issue with Android devices, ios one is working fine.
I think it's related to codec or view element.

**Comment by praveshkhatana** [comment]
please share your updates

**Comment by mouf13** [comment]
@praveshkhatana  https://we.tl/t-12Z5ebNGWh

**Comment by praveshkhatana** [comment]
thanks @mouf13 :)

you can have a look at mine, if you need camera flip option
https://github.com/praveshkhatana/nativescript-twilio-video

**Comment by mouf13** [comment]
Thx @praveshkhatana i will

**Comment by praveshkhatana** [comment]
Hey @mouf13 

Did you find a working solution for Android? Video feed is not working even in between two android devices. :(

**Comment by mouf13** [comment]
hi @praveshkhatana sorry for my late answer, we finally switch to reactnative, works well on ios&android.

**Comment by armackey** [comment]
doesn't look like there's an issue here but a request.

**Comment by armackey** [comment]
Will implement codecs soon.
